src/ros2_aruco/ros2_aruco/ros2_aruco/aruco_node.py

```diff
@@ -65,31 +61,45 @@ def __init__(self):
         )
 
         self.declare_parameter(
-            name="image_topic",
-            value="/camera/image_raw",
+            name="poll_delay",
```

Comment:
I know it is mentioned above but I think you should include unit of time used for delay (ex: poll_delay_sec)